### PR TITLE
GBM Announcement

### PR DESCRIPTION
**Pull request by dejazzhands**
Hi Chase, Bharat, or Christian,

here's the announcement and it's ready to go on the blog.

### CODE REVIEW

File: blog/2021-06-09 GBM announcement.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#ACM General Body Meeting Announcement
```
Comment (clee231) [inline blog/2021-06-09 GBM announcement.md:1]: We are missing the frontmatter on this post. Look at some of the other pages for reference on what you should add... but here's my suggestion:
```suggestion
---
slug: gbm-fall-2021
title: ACM General Body Meeting Announcement
author: Sam Effendy
author_title: Secretary 2021-2022 @ ACM@UIC
author_url: https://github.com/dejazzhands
author_image_url: https://avatars.githubusercontent.com/u/49312226?s=50&v=4
tags: ["general-body-meeting", "events", "SIGs"]
---
```

File: blog/2021-06-09 GBM announcement.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Hi ACM members,
+
+It is my pleasure to announce our ACM General Body Meeting, which will be held on Friday Sep 10th at 5PM. You'll get to learn about all the exciting events we planned for this semester, as well as meet SIG leaders and learn what they are about.
```
Comment (clee231) [inline blog/2021-06-09 GBM announcement.md:5]: May I suggest bolding the dates/times?

File: blog/2021-06-09 GBM announcement.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+The meeting will be in hybrid format, held in the CS Lounge (Science and Engineering Labs East, 2nd floor) and Discord simultaneously.
+
+This Wednesday is the engineering org fair in the Maker Space parking lot from 12-4PM. Please make time to come by and say hi. Some orgs will also be doing demonstrations, so it will be a fun experience.
```
Comment (clee231) [inline blog/2021-06-09 GBM announcement.md:9]: I think Christian added a SIG form on DIscord. Should we add that verbiage here too?

**Review by clee231 — CHANGES_REQUESTED** [review]
Please hyphenate the markdown filename. Otherwise, it won't show up on the blog.

File: blog/2021-06-09 GBM announcement.md (hunk 2 above)
Comment (dejazzhands) [reply blog/2021-06-09 GBM announcement.md]: done

File: blog/2021-06-09 GBM announcement.md (hunk 3 above)
Comment (dejazzhands) [reply blog/2021-06-09 GBM announcement.md:17]: i'm not sure... i'll try doing that

**Review by clee231 — APPROVED** [review]

**Comment by clee231** [comment]
@bmiddha  - It looks like this CI check won't ever pass on forked branches...  Do you think we need to add a condition to only match a specific organization? Or maybe to not fail the pipeline when Azure deployments don't work?

**Comment by bmiddha** [comment]
> @bmiddha - It looks like this CI check won't ever pass on forked branches... Do you think we need to add a condition to only match a specific organization? Or maybe to not fail the pipeline when Azure deployments don't work?

we can try enabling the `skip_deploy_on_missing_secrets` setting suggested in the logs
> The deployment_token is required for deploying content. If you'd like to continue the run without deployment, add the configuration skip_deploy_on_missing_secrets set to true in your workflow file

**Comment by bmiddha** [comment]
@clee231 we can try enabling the `skip_deploy_on_missing_secrets` setting suggested in the logs
> The deployment_token is required for deploying content. If you'd like to continue the run without deployment, add the configuration skip_deploy_on_missing_secrets set to true in your workflow file

**Review by bmiddha — APPROVED** [review]

**Pull request merged**